components/ResultViewList.tsx

```diff
@@ -89,7 +89,7 @@ const ResultViewList = ({
               )}
             </li>
           )}
-          {displayCourseOfStudy && (
+          {displayCourseOfStudy && displayCourseOfStudy !== "no info" && (
```

Comment:
I think `"no info"` comes from Mentorship division's informal tagging, and in fact is not a reserved keyword:

![image](https://github.com/AdvisorySG/mentorship-page/assets/2070423/0bfa3428-4957-48fb-9249-6c0c1d37ccfc)

I was under the perception that the text came from a placeholder set in our codebase, but as it turns out our code already accounts for the possibility of missing `courseOfStudy` or `school` and doesn't display these fields... Since it's an issue with the Airtable database, I will be closing this PR and cancelling the associated issue.